File: CooCooBase/Models/Core Data/WalletContents+CoreDataProperties.m
//
//  WalletContents+CoreDataProperties.m
//  
//
//  Created by Omniwyse on 3/6/18.
//
//

#import "WalletContents+CoreDataProperties.h"

@implementation WalletContents (CoreDataProperties)

+ (NSFetchRequest<WalletContents *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"WalletContents"];
}

@dynamic agencyId;
@dynamic allowInteraction;
@dynamic balance;
@dynamic descriptation;
@dynamic designator;
@dynamic fare;
@dynamic group;
@dynamic identifier;
@dynamic instanceCount;
@dynamic member;
@dynamic purchasedDate;
@dynamic expirationDate;
@dynamic slot;
@dynamic status;
@dynamic ticketActivationExpiryDate;
@dynamic ticketEffectiveDate;
@dynamic ticketExpiryDate;
@dynamic ticketGroup;
@dynamic ticketIdentifier;
@dynamic type;
@dynamic valueOriginal;
@dynamic valueRemaining;
@dynamic activationCount;
@dynamic ticketSource;
@dynamic farecodeExpiryDateTime;

@end

File: Genfare/ManagedObjects/WalletContents+CoreDataProperties.swift
//
//  WalletContents+CoreDataProperties.swift
//  Genfare
//
//  Created by vishnu on 25/01/19.
//  Copyright © 2019 Omniwyse. All rights reserved.
//
//

import Foundation
import CoreData


extension WalletContents {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<WalletContents> {
        return NSFetchRequest<WalletContents>(entityName: "WalletContents")
    }

    @NSManaged public var activationCount: NSNumber?
    @NSManaged public var activationDate: NSNumber?
    @NSManaged public var agencyId: NSNumber?
    @NSManaged public var allowInteraction: NSNumber?
    @NSManaged public var balance: String?
    @NSManaged public var descriptation: String?
    @NSManaged public var designator: NSNumber?
    @NSManaged public var expirationDate: String?
    @NSManaged public var fare: NSNumber?
    @NSManaged public var generationDate: NSNumber?
    @NSManaged public var group: String?
    @NSManaged public var identifier: String?
    @NSManaged public var instanceCount: NSNumber?
    @NSManaged public var member: String?
    @NSManaged public var purchasedDate: NSNumber?
    @NSManaged public var slot: NSNumber?
    @NSManaged public var status: String?
    @NSManaged public var ticketActivationExpiryDate: NSNumber?
    @NSManaged public var ticketEffectiveDate: NSNumber?
    @NSManaged public var ticketExpiryDate: NSNumber?
    @NSManaged public var ticketGroup: String?
    @NSManaged public var ticketIdentifier: String?
    @NSManaged public var ticketSource: String?
    @NSManaged public var type: String?
    @NSManaged public var valueOriginal: NSNumber?
    @NSManaged public var valueRemaining: NSNumber?

}
